src/ldf-client-ui.js

```diff
@@ -673,9 +711,102 @@ require('yasgui-yasqe/dist/yasqe.css'); // Make webpack import the css as well
       if (this._writeResult) {
         this._resultCount++;
         this._writeResult(result);
+
+        // Adding wkt points on a map
+        for (const value in result) {
+          // This code adds points to the map
+          var checkingResult = `${result[value]}`;
+          if (checkingResult.includes('http://www.openlinksw.com/schemas/virtrdf#Geometry') || checkingResult.includes('http://www.opengis.net/ont/geosparql#wktLiteral')) {
+            let wktIncludingVariable = `${result[value]}`;
```

Comment:
```suggestion
            let wktVariable = `${result[value]}`;
```

Reply:
I did work with changing the checkingResult variable on line 718 to wktIncludingVariable and did away with the line 720